Source/WTF/wtf/BitVector.h

```diff
@@ -470,10 +470,10 @@ WTF_ALLOW_UNSAFE_BUFFER_USAGE_END
         size_t numBits() const { return m_numBits; }
         size_t numWords() const { return (m_numBits + bitsInPointer() - 1) / bitsInPointer(); }
 
-        std::span<const uint8_t> byteSpan() const LIFETIME_BOUND { return unsafeMakeSpan(reinterpret_cast<const uint8_t*>(bits()), byteCount(numBits())); }
-        std::span<uint8_t> byteSpan() LIFETIME_BOUND { return unsafeMakeSpan(reinterpret_cast<uint8_t*>(bits()), byteCount(numBits())); }
-        std::span<const uintptr_t> wordsSpan() const LIFETIME_BOUND { return unsafeMakeSpan(bits(), numWords()); }
-        std::span<uintptr_t> wordsSpan() LIFETIME_BOUND { return unsafeMakeSpan(bits(), numWords()); }
+        std::span<const uint8_t> byteSpan() const LIFETIME_BOUND { return unsafeMakeSpan(reinterpret_cast<const uint8_t*>(m_words), byteCount(numBits())); }
+        std::span<uint8_t> byteSpan() LIFETIME_BOUND { return unsafeMakeSpan(reinterpret_cast<uint8_t*>(m_words), byteCount(numBits())); }
```

Comment:
The old code used bit casts and this uses reinterpret casts. Is there a reason to choose one over the other?

Reply:
The code I removed was using bit_cast whereas the code I'm updating was using reinterpret_cast and still is. I did not purposely change the way of casting. I can probably use std::bit_cast here since we're casting to bytes.

Reply:
Actually using `std::bit_cast` doesn't build here.